invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositResults.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2020 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import {
+  LayoutSwitcher,
+  Pagination,
+  ResultsMultiLayout,
+  Sort,
+} from "react-searchkit";
+import { Button, Input, Checkbox, Grid, Segment } from "semantic-ui-react";
+
+export class Results extends Component {
+  render() {
+    const { sortValues, currentResultsState } = this.props;
+    const { total } = currentResultsState.data;
+    return (
+      total && (
+        <>
+          <Input icon="search" placeholder="Search uploads..." />
+          <Button
+            color="green"
+            icon="upload"
+            floated="right"
+            href="/deposits/new"
+            content="New upload"
+          />
+          <Segment>
+            <Grid>
+              <Grid.Row verticalAlign="middle" className="header-row">
+                <Grid.Column width={7}>
+                  <Checkbox label="Select all" />
+                </Grid.Column>
+                <Grid.Column width={5} textAlign="right">
+                  <LayoutSwitcher defaultLayout="list" />
+                </Grid.Column>
+                <Grid.Column width={4} textAlign="right">
+                  {sortValues && (
+                    <Sort
+                      values={sortValues}
+                      label={(cmp) => <>Sort by {cmp}</>}
+                    />
```

Comment:
The reason why there's a lot of code repetition in this PR is that this part of `Results.js` is custom and different from the main search. There's no overridable mechanism here so I had to create a new SearchApp.js (`RDMDepositSearchApp.js`) that imports this file and bundles it in a new `invenio-app-rdm-deposits-search-app-js.js` file. If you know any other way to avoid this repetition please let me know.